app/assets/stylesheets/modules/gem.css

```diff
@@ -31,6 +31,11 @@
       overflow: auto;
       border-bottom: 1px solid #c1c4ca; } }
 
+.delete__owners{
```

Comment:
Thanks. I didn't know about BEM. I have changed `delete_owners` to `owners` block. My reasoning is mostly visual:
![screenshot from 2016-07-14 21-58-58](https://cloud.githubusercontent.com/assets/7680662/16873924/db1b1662-4ab3-11e6-9763-3172e53a9b03.png)